src/database/db_update.js

```diff
@@ -5,6 +5,15 @@ const updateLikes = (likeNum, memoryId, cb) => {
     'UPDATE memories SET likes = $1 WHERE memories.id = $2', [likeNum, memoryId], cb);
 };
 
+const updatePhotoMemory = (userId, tag, heading, imageId, cb) => {
+  connect.query(
+    `UPDATE memories
+     SET tag = $2, heading = $3
+     WHERE user_id = (SELECT users.id FROM users WHERE username = $1 OR email = $1)
```

Comment:
```sql
WHERE user_id = (SELECT users.id FROM users WHERE username = $1 OR email = $1)
```
is the same as
```sql
WHERE username = $1 OR email = $1
```
?

I am fairly confused, this set the heading and tag of a memory, using it image id and user id?